build/azure-pipelines.yml

```diff
@@ -0,0 +1,46 @@
+name: Astro Panda Blazor Power BI
+
+variables:
+- name: library
+  value: Blazor Power BI
+- name: targetProject
+  value: "**/AstroPanda.Blazor.PowerBI.csproj"
+
+trigger:
+- main
+
+pr:
+  branches:
+    include:
+    - '*'
+
+jobs:
+- job: Build
+  pool:
+    vmImage: windows-latest
+  steps:
+  - task: DotNetCoreCli@2
+    displayName: Restore ${{ variables.library }}
+    inputs:
+      command: build
+      projects: ${{ variables.targetProject }}
+
+  # - task: DotNetCoreCli@2
```

Comment:
This could be removed if you're done testing the pipeline.

Reply:
Well I want to come back and add tests soon, the pipeline would fail without a test project right now